src/Craned/JobManager.cpp

```
@@ -0,0 +1,173 @@
+  info.cgroup_exists = false;
+
+  if (auto task_ids = this->m_uid_to_job_ids_map_[uid]; task_ids) {
+    if (!task_ids) {
+      CRANE_WARN("Uid {} not found in uid_to_task_ids_map", uid);
+      return std::nullopt;
+    }
+    info.cgroup_exists = true;
+    info.job_cnt = task_ids->size();
+    info.first_task_id = *task_ids->begin();
+  }
+  return info;
```

Cppcheck (2.10-2)

[warning] Uninitialized variable

(uninitvar)